motr/client_internal.h

```diff
@@ -464,8 +469,15 @@ enum {
 	 * These constants are used to create buffers acceptable to the
 	 * network code.
 	 */
+	
+
```

Comment:
Remove at least one empty space here.

Reply:
Removed one line out of the two to maintain legibility